bazel/repositories.bzl

```diff
@@ -0,0 +1,86 @@
+# The build rules below for external dependencies build rules are maintained
+# on a best effort basis. The rules are provided for developer convenience. For production builds,
+# we recommend building the libraries according to their canonical build systems and expressing the
+# dependencies in a manner similar to ci/WORKSPACE.
+
+def googletest_repositories():
+    BUILD = """
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+
+cc_library(
+    name = "googletest",
+    srcs = [
+        "googlemock/src/gmock-all.cc",
+        "googletest/src/gtest-all.cc",
+    ],
+    hdrs = glob([
+        "googletest/include/**/*.h",
+        "googlemock/include/**/*.h",
+        "googletest/src/*.cc",
+        "googletest/src/*.h",
+        "googlemock/src/*.cc",
+    ]),
```

Comment:
Style: `glob([])` should be sorted alphabetically, so everything from `googlemock/` should go before `googletest/`.